lib/pbench/client/oidc_admin.py

```diff
@@ -8,7 +8,7 @@
 
 
 class OIDCAdmin(Connection):
-    OIDC_REALM = "pbench-server"
+    OIDC_REALM = os.getenv("OIDC_REALM", "pbench-server")
     ADMIN_USERNAME = os.getenv("ADMIN_USERNAME", "admin")
     ADMIN_PASSWORD = os.getenv("ADMIN_PASSWORD", "admin")
```

Comment:
I'm thinking that the names `ADMIN_USERNAME` and `ADMIN_PASSWORD` might be a little too generic, considering they come from the general shell environment.  `OIDC_ADMIN_USERNAME` would be better.  (Or, failing that, `PBENCH_*`....)